api/src/main/java/org/openmrs/api/InvalidTokenException.java

```diff
@@ -15,37 +15,27 @@
  * Represents often fatal errors that occur when a user cannot be found due to incorrect, expired
  * token or wrong email and or username.
  */
-public class UserNotFoundException extends APIException {
+public class InvalidTokenException extends APIException {
```

Comment:
I didn't literally  say name it InvalidTokenException, to be exact it is an Invalid activation key so it should be InvalidActivationKeyException